dc-cudami-admin/dc-cudami-admin-webapp/src/main/java/de/digitalcollections/cudami/admin/controller/identifiable/entity/DigitalObjectsController.java

```diff
@@ -81,24 +68,22 @@ public String edit(@PathVariable UUID pathUuid, @ModelAttribute @Valid DigitalOb
   @RequestMapping(value = "/digitalobjects", method = RequestMethod.GET)
   public String list(Model model, @PageableDefault(sort = {"lastModified"}, size = 25) Pageable pageable) {
     final PageRequest pageRequest = PageableConverter.convert(pageable);
-    final PageResponse pageResponse = digitalObjectService.find(pageRequest);
+    final PageResponse pageResponse = service.find(pageRequest);
     Page page = PageConverter.convert(pageResponse, pageRequest);
-    model.addAttribute("defaultLocale", localeService.getDefaultLocale());
     model.addAttribute("page", new PageWrapper(page, "/digitalobjects"));
     return "digitalobjects/list";
   }
 
   @RequestMapping(value = "/digitalobjects/{uuid}", method = RequestMethod.GET)
   public String view(@PathVariable UUID uuid, Model model) {
-    DigitalObject digitalObject = (DigitalObject) digitalObjectService.get(uuid);
+    DigitalObject digitalObject = (DigitalObject) service.get(uuid);
     model.addAttribute("availableLocales", digitalObject.getLabel().getLocales());
-    model.addAttribute("defaultLocale", localeService.getDefaultLocale());
     model.addAttribute("digitalObject", digitalObject);
     return "digitalobjects/view";
   }
 
   // ----------------------------------------------------------------------------
   public void setWebsiteService(DigitalObjectService digitalObjectService) {
-    this.digitalObjectService = digitalObjectService;
+    this.service = digitalObjectService;
```

Comment:
Nit: Wird der Setter vom Code irgendwo verwendet oder ist das nur für die Spring DI notwendig? Falls letzteres, vielleicht auf Constructor-Injection wechseln und auf das `@Autowired` auf dem Feld und diesen Setter verzichten.

Reply:
Ich glaube nicht, in den ganzen anderen Controllern gibt es auch nichts in der Art.

Reply:
Falls die Usage Search von deiner IDE auch nichts gibt würde ich's wie vorgeschlagen machen, Constructor Injection und setter und @Autowired auf dem Feld entfernen.

Reply:
Hab an vielen Stellen auf Constructor Injection umgestellt.